src/resources/projectMSDL.properties.in

```diff
@@ -145,9 +140,13 @@ logging.channels.async.class = AsyncChannel
 logging.channels.async.channel = split
 
 # Default logging settings.
-logging.loggers.root.level = information
 logging.loggers.root.channel = async
 
+# Set log level to debug for all components
+#logging.loggers.root.level = debug
+logging.loggers.root.level = information
+
+
```

Comment:
Should remove this addition here before merging, was probably a leftover from testing.
```suggestion
```